### PR TITLE
Create project symlink to installed directory

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**
@harithaIBM to test with lz4/xxHash

### CODE REVIEW

File: bin/lib/zopen-build [hunk 1]
```diff
@@ -840,6 +843,11 @@ resolveCommands()
     export ZOPEN_INSTALL_CMD="${ZOPEN_INSTALL} ${ZOPEN_INSTALL_OPTS}"
     paxFileName="${ZOPEN_NAME}.${LOG_PFX}.zos.pax.Z"
     export ZOPEN_PAX_CMD="pax -w -z -x pax \"-s#${ZOPEN_INSTALL_DIR}/#${ZOPEN_NAME}.${LOG_PFX}.zos/#\" -f \"${paxFileName}\" \"${ZOPEN_INSTALL_DIR}/\""
+    PROJECT_NAME="$(echo ${ZOPEN_NAME} | cut -d'-' -f1)"
+    if [ "${PROJECT_NAME}" != "${ZOPEN_NAME}" ]; then
+      linkPath="$(dirname ${ZOPEN_INSTALL_DIR})/${PROJECT_NAME}"
+      export ZOPEN_LINK_CMD="rm -f ${linkPath} && ln -s ${ZOPEN_INSTALL_DIR} ${linkPath}"
```
Comment (MikeFultonDev) [inline bin/lib/zopen-build]: would it make sense to verify it's a symbolic link before removing it, in case it is a directory of stuff?

Comment (IgorTodorovskiIBM) [reply bin/lib/zopen-build]: Should we introduce a --clobber option to clobber the symlink/ZOPEN_INSTALL_DIR?

Comment (IgorTodorovskiIBM) [reply bin/lib/zopen-build]: I've added a check for now

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM does this still need to be merged?

**Comment by IgorTodorovskiIBM** [comment]
Closing in favor of https://github.com/ZOSOpenTools/utils/pull/129

**Pull request closed without merging**